assets/locales/en.json

```
@@ -141,5 +135,27 @@
   "runGarbageCollectorDone": {
     "title": "Garbage collector",
     "message": "The garbage collector ran successfully."
+  },
+  "downloadCidContentDialog": {
+    "title": "Download",
+    "message": "Type down a CID, IPFS or IPNS path whose contents you want to download.",
+    "action": "Download"
+  },
+  "cantResolveCidDialog": {
+    "title": "Could not resolve",
+    "message": "The given path ({ path }) could not be resolved."
```

```suggestion
    "message": "Unable to resolve \"{ path }\"."
```
Probably want to avoid the use of the word "path" here in case someone pastes in a CID.